diff --git a/node_modules/@mui/material/esm/styles/cssUtils.js b/node_modules/@mui/material/esm/styles/cssUtils.js
index fa396b2..289512c 100644
--- a/node_modules/@mui/material/esm/styles/cssUtils.js
+++ b/node_modules/@mui/material/esm/styles/cssUtils.js
@@ -106,7 +106,8 @@ export function responsiveProperty({
   max,
   unit = 'rem',
   breakpoints = [600, 900, 1200],
-  transform = null
+  transform = null,
+  mediaUnit = 'px'
 }) {
   const output = {
     [cssProperty]: `${min}${unit}`
@@ -117,7 +118,7 @@ export function responsiveProperty({
     if (transform !== null) {
       value = transform(value);
     }
-    output[`@media (min-width:${breakpoint}px)`] = {
+    output[`@media (min-width:${breakpoint}${mediaUnit})`] = {
       [cssProperty]: `${Math.round(value * 10000) / 10000}${unit}`
     };
   });
diff --git a/node_modules/@mui/material/esm/styles/responsiveFontSizes.js b/node_modules/@mui/material/esm/styles/responsiveFontSizes.js
index 61f74a3..a2907c3 100644
--- a/node_modules/@mui/material/esm/styles/responsiveFontSizes.js
+++ b/node_modules/@mui/material/esm/styles/responsiveFontSizes.js
@@ -59,7 +59,8 @@ export default function responsiveFontSizes(themeInput, options = {}) {
         max: maxFontSize,
         unit: 'rem',
         breakpoints: breakpointValues,
-        transform
+        transform,
+        mediaUnit: theme.breakpoints?.unit || 'px'
       })
     };
   });
