diff --git a/node_modules/drizzle-orm/pg-core/columns/timestamp.cjs b/node_modules/drizzle-orm/pg-core/columns/timestamp.cjs
index 207a6df..80ec01b 100644
--- a/node_modules/drizzle-orm/pg-core/columns/timestamp.cjs
+++ b/node_modules/drizzle-orm/pg-core/columns/timestamp.cjs
@@ -59,7 +59,7 @@ class PgTimestamp extends import_common.PgColumn {
     return value;
   }
   mapToDriverValue = (value) => {
-    return value.toISOString();
+    return value instanceof Date ? value.toISOString() : value;
   };
 }
 class PgTimestampStringBuilder extends import_date_common.PgDateColumnBaseBuilder {
diff --git a/node_modules/drizzle-orm/pg-core/columns/timestamp.js b/node_modules/drizzle-orm/pg-core/columns/timestamp.js
index 6ce67ac..ec77d8b 100644
--- a/node_modules/drizzle-orm/pg-core/columns/timestamp.js
+++ b/node_modules/drizzle-orm/pg-core/columns/timestamp.js
@@ -32,7 +32,7 @@ class PgTimestamp extends PgColumn {
     return value;
   }
   mapToDriverValue = (value) => {
-    return value.toISOString();
+    return value instanceof Date ? value.toISOString() : value;
   };
 }
 class PgTimestampStringBuilder extends PgDateColumnBaseBuilder {
